mavedb_mapping/transcript_selection_helper.py

```diff
@@ -97,3 +97,33 @@ def get_clingen_id(hgvs):
         return page.split("/")[4]
     except:
         return "NA"
+
+
+def check_non_human(mave_blat_dict, min_percentage=80):
```

Comment:
You may consider returning a bool instead of a string for this function, and rename the function to something like `is_human` instead of `check_non_human`. I know you are also considering a refactor to object-oriented design; this function seems to be oriented around output from another function, which is an indicator the functions (and perhaps some associated variables) are appropriate to aggregate into a class (as you have started doing in #16). This will let you do things like:

self.is_human() # returns true if alignment mapping is consistent with a human target sequence

Reply:
I agree with Alex's comment. Returning a bool, true/false, instead of a string for this function is better if this function is used in checking a sample is human or not. It will be easier to handle when you call this function from other functions.